db/models.py

```diff
@@ -1,4 +1,11 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
+from django.utils.timezone import now
+
+
+class User(AbstractUser):
+    pass
```

Comment:
The `User` model extends `AbstractUser` without adding new fields or methods. Consider using Django's default `User` model unless you plan to extend it with additional functionality.